### PR TITLE
[ADD] website_compress_html

### PR DESCRIPTION
**Pull request by Benniphx**
added the option to activate the HTML compession from the master for version 8

### CODE REVIEW

File: website_compress_html/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+Compress HTML Code Output
+=========================
+
+added the option to activate the HTML compession from the master for version 8
```
Comment (pedrobaeza) [inline website_compress_html/README.md:4]: Please start sentences in upper

Comment (pedrobaeza) [inline website_compress_html/README.md:4]: s/compession/compression

Comment (pedrobaeza) [inline website_compress_html/README.md:4]: s/from the master for/backported from Odoo master version to

File: website_compress_html/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+you can turn it on or off in the backend
```
Comment (pedrobaeza) [inline website_compress_html/README.md:6]: Please put a section "Configuration" instead explaining full path to configure this.

Comment (pedrobaeza) [inline website_compress_html/README.md:6]: You can also add a section usage with the new "Try me on runbot" button.

File: website_compress_html/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+Credits
+=======
+
+Contributors
+------------
+
+* Benjamin Bachmann (benniphx@gmail.com)
```
Comment (pedrobaeza) [inline website_compress_html/README.md:14]: You should put here Odoo and try to identify which people inside Odoo.

File: website_compress_html/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "Compress HTML",
+    'summary': """Compress HTML Code Output""",
+    'description': """Compress HTML Code Output""",
```
Comment (pedrobaeza) [inline website_compress_html/__openerp__.py:24]: Don't add this key, as you have already added the README file

File: website_compress_html/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    'author': "bloopark systems GmbH & Co. KG, "
```
Comment (pedrobaeza) [inline website_compress_html/__openerp__.py:25]: Odoo is not here

File: website_compress_html/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+              "Odoo Community Association (OCA)",
+    'website': "http://www.bloopark.de",
+    'license': 'AGPL-3',
```
Comment (pedrobaeza) [inline website_compress_html/__openerp__.py:28]: Are you sure this code is AGPL-3. I think it's LGPL-3

File: website_compress_html/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+    'category': 'Speed',
+    'version': '1.0',
```
Comment (pedrobaeza) [inline website_compress_html/__openerp__.py:30]: Use new version numbering 8.0.1.0.0

**Comment by pedrobaeza** [comment]
You should extract commit history from Odoo codebase with `git filter-branch` and then make the needed changes attributing yourself, but you can't overwrite authorship this way. You can see an example of how to do it here: https://github.com/odoomrp/odoomrp-wip/wiki/Migration-to-OCA

**Comment by rafaelbn** [comment]
Hi @Benniphx are you going to continue with this PR? Let me know please. If not are you agree to close it? Thanks!

**Comment by simahawk** [comment]
@Benniphx no answer since more than 2yrs.
Closing this but feel free to re-open it if needed, thanks!

**Pull request closed without merging**